dockerfiles/Dockerfile

```diff
@@ -121,3 +121,10 @@ COPY --from=ser2net /opt/ser2net /
 VOLUME /opt/conf
 
 CMD ["/entrypoint.sh"]
+
+#
+# SFTP server
+#
+FROM alpine:3.17 as labgrid-sftp-server
+
+RUN apk add openssh-sftp-server
```

Comment:
It seems that the podman in Ubuntu 22.04 tries to build the other targets as well, so this should be a different Dockerfile.